src/components/Card/CardSkeleton/CardSkeleton.js

```diff
@@ -3,39 +3,54 @@
  * @copyright IBM Security 2019
  */
 
+import classnames from 'classnames';
+import { PropTypes } from 'prop-types';
```

Comment:
This doesn't throw any errors so it doesn't seem like a big issue, but elsewhere we use either `import PropTypes from 'prop-types';` or `import { string } from 'prop-types';` - what's your preference?

Reply:
Oh sure thing, I'll remove the brackets.